decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -169,10 +178,30 @@ def profiling_gems
         copy_file "rack_profiler_initializer.rb", "config/initializers/rack_profiler.rb"
       end
 
+      def tweak_spring
+        run "bundle exec spring binstub --all"
+
+        create_file "config/spring.rb", <<~CONFIG
+          require "decidim/spring"
+
+          Spring.watch(
+            ".ruby-version",
+            ".rbenv-vars",
+            "tmp/restart.txt",
+            "tmp/caching-dev.txt"
+          )
+        CONFIG
+      end
+
       def bundle_install
         run "bundle install"
       end
 
+      def copy_migrations
```

Comment:
Moving this as the last step, so you don't need to wait for the whole seeds to finish running to have the things that were after this (like letter_opener)